src/sagemaker/model_uris.py

```diff
@@ -28,6 +28,8 @@ def retrieve(
     model_id=None,
     model_version: Optional[str] = None,
     model_scope: Optional[str] = None,
+    tolerate_vulnerable_model: Optional[bool] = None,
+    tolerate_deprecated_model: Optional[bool] = None,
```

Comment:
remove `Optional` and use `False` as default please.

Reply:
I'd rather keep the pattern of setting these params to `None`.